service/amazonsns/amazon_sns.go

```diff
@@ -11,6 +11,8 @@ import (
 )
 
 // SNSSendMessageAPI Basic interface to send messages through SNS.
+//
+//go:generate mockery --name=SNSSendMessageAPI --output=. --case=underscore --inpackage
 type SNSSendMessageAPI interface {
```

Comment:
I just noticed that the types for `Amazon SNS` are all exported, even though only one needs to be exported. I'll quickly fix this and push it. Interesting that we're only noticing now.

Reply:
Should be fixed by 3c51987712b8f7196936d7369243be7c45979a39.